test/api/TestFactory.js

```diff
@@ -84,12 +84,12 @@ window.TestFactory.prototype.exposeStorageActors = function() {
   this.logger.info('- exposeStorageActors');
   return Promise.resolve()
     .then(() => {
-      TestFactory.storage_service = new z.storage.StorageService();
+      TestFactory.storage_service = singleton(z.storage.StorageService);
```

Comment:
Do we want to de-snake_case it as well?

Reply:
possibly, I don't really have an opinion on this @bennyn @gregor ?

Reply:
We should not refactor the `TestFactory` in this PR.